Updated the Quicksilver Doc to Include a  Note emphasizing the Correct … (#7017)
pantheon-systems/documentation: source/content/quicksilver.md

## Script Type and Location

> Quicksilver currently supports `webphp` scripting, which runs a PHP script through the same runtime environment as the website itself. PHP scripts are subject to the same limits as any code on the platform, such as [timeouts](/timeouts). PHP scripts cannot be batched, and will run in order, with each command executing after the previous command has finished or timed out. In the future we may add additional types. 

Reviewer: I'm not sure, but I don't think we should mention things that we don't currently offer (and are not currently working on) such as "In the future we may add additional types. "

how about?...

```suggestion
Quicksilver currently supports `webphp` scripting, which runs a PHP script through the same runtime environment as the website. PHP scripts are subject to the same limits as any code on the platform, such as [timeouts](/timeouts). PHP scripts cannot be batched, and run continuously and sequentially. Each command executes after the previous command has finished or timed out.
```

Setting up a dedicated directory in the docroot (for example, `private/scripts`) for tracking these files is recommended. Setting the `web_docroot` to `true` instructs Quicksilver to look for files inside the `web` folder. If your site uses a [nested docroot](/nested-docroot), the scripts directory must be located in the `web` subdirectory of your site's code repository (for example, `web/private/scripts`). 

<Alert type="info" title="Note">

The script paths in your `pantheon.yml` file should not include the `web/` path prefix. Scripts in your `pantheon.yml` file should match the following path examples: `private/scripts/new_relic_deploy.php` or `private/scripts/slack_deploy_notification.php`

> </Alert>

Author: @dficker Thanks! I will take a closer look at this on Monday. Have a great weekend!
